tsl/src/data_node.c

```diff
@@ -857,18 +857,37 @@ data_node_add_internal(PG_FUNCTION_ARGS, bool set_distid)
 Datum
 data_node_add(PG_FUNCTION_ARGS)
 {
+#if PG16_GE
+	ereport(ERROR,
+			(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+			 errmsg("adding data node is not supported"),
+			 errhint("Multi-node is not supported anymore on PostgreSQL >= 16.")));
```

Comment:
Again, hints should give an idea of how you can "fix" the problem, so suggest turning the hint into details in this case:
```suggestion
	ereport(ERROR,
			(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
			 errmsg("adding data node is not supported"),
			 errdetail("Multi-node is not supported anymore on PostgreSQL >= 16.")));
```

Reply:
Ooops... fixed!